Spigot-Server-Patches/0551-Optimize-Villagers.patch

```diff
@@ -93,25 +105,40 @@ index 35eb3a5a61145e94d5b0c77c0eb13bfa46fac23b..6861b1a345496a83900b0ef702ba3431
 +                }
 +            }
 +        }
-+        // Clean up - vanilla does this only when it runs out, but that would push it to try farther POI's...
-+        this.e.long2LongEntrySet().removeIf((entry) -> entry.getLongValue() < this.d);
++
+         this.f.long2ObjectEntrySet().removeIf((entry) -> {
+-            return !((BehaviorFindPosition.a) entry.getValue()).b(i);
++            return entry.getValue().b < e;
+         });
 +        /*
          Predicate<BlockPosition> predicate = (blockposition) -> {
-             long j = blockposition.asLong();
+             BehaviorFindPosition.a behaviorfindposition_a = (BehaviorFindPosition.a) this.f.get(blockposition.asLong());
  
-@@ -61,6 +108,6 @@ public class BehaviorFindPosition extends Behavior<EntityCreature> {
-                 return entry.getLongValue() < this.d;
-             });
+@@ -96,6 +141,7 @@ public class BehaviorFindPosition extends Behavior<EntityCreature> {
+                 });
+             }
          }
--
-+        */ // Paper end
++         */ // Paper end
```

Comment:
9 spaces?

Reply:
Um wtf happened here? I remember intellij autocompleting that

Reply:
Intellisense doesn't apply to whitespace...

Reply:
bruh
whatever now's been fixed